server.js

```diff
@@ -51,34 +52,55 @@ io.sockets.on("connection", socket => {
     socket.to(broadcaster).emit("disconnectPeer", socket.id);
   });
 
+  socket.on("streamerTimeout", () => {
+    console.log("Streamer closed the window, waiting for 10 minutes to recover")
```

Comment:
The point is that when the streamer window closes, 10 minute timer starts. After this, server orders broadcaster to close connection and viewers to refresh page so the poster pic appears